app/models/metadata.rb

```diff
@@ -26,6 +26,64 @@ class Metadata < ApplicationRecord
     }
   ].to_json
 
+  def self.filters_to_json
```

Comment:
Can this be moved into a serializer if the only intent is serving the data to the front-end?
Ideally the method will receive the data as input